osquery/tables/system/posix/authorized_keys.cpp

```diff
@@ -7,25 +7,45 @@
  * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
  */
 
-#include <string>
 #include <vector>
 
 #include <osquery/core/core.h>
 #include <osquery/core/tables.h>
 #include <osquery/filesystem/filesystem.h>
-#include <osquery/logger/logger.h>
+
+#include <osquery/tables/system/posix/authorized_keys.h>
 #include <osquery/tables/system/system_utils.h>
+#include <osquery/utils/conversions/join.h>
 #include <osquery/utils/conversions/split.h>
 #include <osquery/utils/system/system.h>
 #include <osquery/worker/ipc/platform_table_container_ipc.h>
-#include <osquery/worker/logging/glog/glog_logger.h>
 
 namespace osquery {
 namespace tables {
 
 const std::vector<std::string> kSSHAuthorizedkeys = {".ssh/authorized_keys",
                                                      ".ssh/authorized_keys2"};
 
+const std::vector<std::string> kSSHKeyTypes = {"ssh-rsa",
```

Comment:
What about DSA? And likely other types. On my mac, I can generate `ssh-dss`. I don't know of the SK types come in differently.

Do we need to enumerate these? It's the kind of thing that I suspect will go out of date.

Reply:
Though reading about things, I suspect it's simplest to enumerate the expected types here, than to try to guess whether something is an option or a key type

Reply:
I've changed the code according to this comment, I hope you'll find it much better now.